src/keyboard/keyboardImplementation.ts

```diff
@@ -65,6 +76,7 @@ export async function keyboardImplementation(
     if (options.delay > 0) {
       await wait(options.delay)
     }
+
     return keyboardImplementation(text.slice(consumedLength), options, state)
```

Comment:
I think the implementation should take place here.
Pushing the `keyDef` that is to be repeated on the state. Something like
```js
state.repeatKey = { times: fireKeydownTimes, consumedLength, keyDef }
```
Then these should be used on the next iteration instead of `getNextKeyDef()`.
And when times is counted down to `0` it should be removed and `consumedLength` should be applied.

Reply:
Changed 👍🏻